src/uu/ls/src/ls.rs

```diff
@@ -1,6 +1,7 @@
 // This file is part of the uutils coreutils package.
 //
 // (c) Jeremiah Peschka <jeremiah.peschka@gmail.com>
+// (c) Robert Swinford <robert.swinford(at)gmail.com>
```

Comment:
please remove this, it isn't adding much value + you will be credited in the git history

Reply:
Yeah, no problem.